frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.tsx

```diff
@@ -415,16 +415,17 @@ export function ClustersTable(props: {
           const unDestroyedClusters = clusters.filter((cluster) => !cluster.isHive)
           setModalProps({
             open: true,
-            alert: (
-              <Alert
-                variant="danger"
-                isInline
-                title={t('You selected {{0}} clusters that cannot be destroyed', [unDestroyedClusters.length])}
-                style={{ marginTop: '20px' }}
-              >
-                <TextContent>{t('They will not be destroyed when you perform this action.')}</TextContent>
-              </Alert>
-            ),
+            alert:
+              unDestroyedClusters.length > 0 ? (
+                <Alert
+                  variant="danger"
+                  isInline
+                  title={t('You selected {{0}} clusters that cannot be destroyed', [unDestroyedClusters.length])}
+                  style={{ marginTop: '20px' }}
+                >
+                  <TextContent>{t('They will not be destroyed when you perform this action.')}</TextContent>
```

Comment:
Then we can change this to:
`The cluster(s) will not be destroyed when you perform this action.`

Since `They` suggest multiple clusters but it might only be one.

Reply:
You can use plural support by calling the interpolation variable `count`. When you run the tooling to update the translations file, it will automatically create 2 entries, one for singular and one for plural. Just edit them to only use the `s` for the plural case.